include/tmp/file

```diff
@@ -44,29 +44,35 @@ public:
     /// Creates a unique temporary binary file
     ///
     /// The file path consists of the system's temporary directory path, the
-    /// given prefix, and random characters to ensure path uniqueness
+    /// given prefix, extension, and random characters to ensure path uniqueness
     ///
-    /// @param prefix   A prefix to be used in the temporary file path
+    /// @param prefix    A prefix to be used in the temporary file path
+    /// @param extension An extension to be used in the temporary file path
     /// @throws std::filesystem::filesystem_error if cannot create a file
-    explicit file(std::string_view prefix = "");
+    explicit file(std::string_view prefix = "",
+                  std::string_view extension = "");
```

Comment:
It's better to call it a suffix since you're not actually checking its contents
```suggestion
                  std::string_view suffix = "");
```

Reply:
Done